packages/ui/src/components/Button/SocialLinkButton.tsx

```diff
@@ -0,0 +1,42 @@
+import { ConnectorMetadata } from '@logto/schemas';
+import classNames from 'classnames';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import * as SocialLinkButtonStyles from './SocialLinkButton.module.scss';
+import * as styles from './index.module.scss';
+
+export type Props = {
+  isDisabled?: boolean;
+  className?: string;
+  connector: Pick<ConnectorMetadata, 'id' | 'name' | 'logo'>;
+  onClick?: (id: string) => void;
+};
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const noop = () => {};
```

Comment:
Is this necessary? `onClick` could just be `undefined`, and we can use `onClick?.()` to call it

Reply:
good call